README.md

```diff
@@ -67,12 +69,12 @@ emulator within your IDE. The emulator allows you configure an environment that
 representative of your service running on Cloud Run.
 
```

Comment:
GitHub won't let me comment on this directly for some reason but in the "Testing functions" section, that information isn't Cloud Function specific, right? Don't those pubsub commands work for Cloud Run, too? Should that be updated?

Also is the "Building a Cloud Run image manually" section still adding value given your new instructions? If so, maybe we can clarify what the differences are? My understanding is that the gcloud build command skips the docker part and just builds it with cloud resources. When would we want to do that process over what you added? Could we elaborate? Also is the TODO in that section (Local docker build instructions) the same as what you added? Does it make sense to collapse that section into what you added?

I also feel like ideally we wouldn't have the Function section and we would rearrange some of these sections and add more background info but I don't really thing it's worth dwelling on it too much for just the prototype. For the real thing though we should :)

Reply:
The manual cloud run build is interchangeable with the docker build and push commands. I'll just remove that section to avoid confusion. I'm going to keep the function section since we still have functions in the repo and it is useful info to have around.